Assets/MixedRealityToolkit/Definitions/CameraSystem/MixedRealityCameraSettingsConfiguration.cs

```diff
@@ -46,6 +46,9 @@ public struct MixedRealityCameraSettingsConfiguration : IMixedRealityServiceConf
         [SerializeField]
         private BaseCameraSettingsProfile settingsProfile;
 
+        /// <inheritdoc />
+        public BaseMixedRealityProfile Profile => settingsProfile;
+
         /// <summary>
         /// Camera settings specific configuration profile.
```

Comment:
Should this be marked `Obsolete`, like you did in MixedRealityServiceConfiguration?